app/web/cms/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
I think this is historical legacy stuff and technically better to be uppercase.

Reply:
According to html standards doc looks like it's case insensitive:

https://html.spec.whatwg.org/multipage/syntax.html#the-doctype

Reply:
Also here's the discussion on this change in the prettier repo, looks like no one reported any bugs:

https://github.com/prettier/prettier/issues/15096

Reply:
OK no worries if it's not causing issues.